osschallenge/tests/test_selenium_logged_in_as_mentor.py

```diff
@@ -0,0 +1,158 @@
+        self.login_page.login("Test", "12345qwert")
+
+        self.user1 = User.objects.create(
+            last_login="2017-10-18 11:55:45.681893+00",
+            is_superuser=False,
+            username="Test",
+            first_name="Test",
+            last_name="Test",
+            email="example@example.ch",
+            is_staff=False,
+            is_active=True,
+            date_joined="2017-10-13 08:17:36.901715+00"
+        )
+        self.user1.set_password("12345qwert")
+        self.user1.save()
+
+        self.role1 = Role.objects.create(
+            id=2,
+            name="Mentor"
+        )
+
+        self.profile1 = Profile.objects.create(
+            user=self.user1,
+            role=self.role1,
+            links="Test",
+            contact="Test",
+            key="Test1",
+            picture="Test.png"
+        )
+
+        self.project1 = Project.objects.create(
+            id=1,
+            title_de="OpenStreetMap",
+            title_en_us="OpenStreetMap",
+            lead_text_de="Blablablab",
+            lead_text_en_us="Blablablab",
+            description_de="Blablablab",
+            description_en_us="Blablablab",
+            licence="MIT",
+            website="www.google.ch",
+            github="www.github.com",
+            owner=self.user1
+        )
+
+        self.task1 = Task.objects.create(
+            id=1,
+            title="Bug Fixing",
+            lead_text="Bug Fixing",
+            description="Bug Fixing",
+            project=self.project1,
+            assignee=None,
+            task_done=False,
+            task_checked=False,
+            picture="test.png",
+            approved_by=None,
+            approval_date="2017-10-18 12:34:51.168157+00"
+        )
+
+        self.client.login(username="Test", password='12345qwert')
+        cookie = self.client.cookies['sessionid']
+        self.driver.get(self.live_server_url + '/login/')
+        self.driver.add_cookie(
+            {
+                'name': 'sessionid',
+                'value': cookie.value,
+                'secure': False,
+                'path': '/'
+            }
+        )
+        self.driver.refresh()
+        self.driver.get(self.live_server_url + '/login/')
+
+    @classmethod
+    def tearDownClass(self):
+        self.driver.quit()
+        super(MydriverTests, self).tearDownClass()
+
+    def test_create_a_new_project(self):
+        self.new_project_page.open()
+        self.new_project_page.create_new_project(
+            "Mein neues Projekt",
+            "My new project",
+            "Einleitung zum Projekt",
+            "Short introduction to the project",
+            "Beschreibung zum Projekt",
+            "Description for the project",
+            "GPL",
+            "www.github.com",
+            "www.example.com"
+        )
+        project = Project.objects.get(title_de="Mein neues Projekt")
+        self.assertEqual(project.title_de, "Mein neues Projekt")
```

Comment:
When you're repeating all the string like "Mein neues Projekt", it's a bit hard to change the test in the future (and also easy to introduce spelling mistakes).
cosider creating a project object at the beginning of the tests, and use the properties of it in `create_new_project` and all the assertions!

Reply:
I changed this. I use now the properties of the objects for the assertions.